features/support/vcr_setup.rb

```diff
@@ -3,9 +3,4 @@
 VCR.configure do |config|
   config.cassette_library_dir = 'features/vcr_cassettes'
   config.allow_http_connections_when_no_cassette = true
-
-  #Capybara: http://stackoverflow.com/a/6120205/1020168
-  config.ignore_request do |request|
-    URI(request.uri).request_uri == "/__identify__"
-  end
```

Comment:
This has been moved to the config used by both RSpec & Cucumber:
[spec/support/shared_vcr_setup.rb](https://github.com/GSA/search-gov/pull/611/files#diff-04c66b80c50320992a4e64851b2ad84b)